rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -920,9 +919,9 @@ def get_form(name):
         for cfg_dict in default_first(self.data):
             if cfg_dict is None:
                 continue
-            cfg, created = ContextFormGroup.objects.get_or_create(
-                registry=registry, name=cfg_dict["name"])
+            cfg, created = ContextFormGroup.objects.get_or_create(code=cfg_dict["code"])
```

Comment:
I got an error when trying to import the migrated export into a blank project:

```
Import Failed: null value in column "registry_id" violates not-null constraint DETAIL: Failing row contains (1, F, , null, D, f, null, C, 1, CFG1). 
```

Previously, registry was included in the `get_or_create`

Reply:
Of course. 
I've added the registry parameter back in.